classification/sampling.csv

```diff
@@ -1,14 +1,22 @@
 Authors,Venue,Year,Title,Algorithm,Input Data,Algorithm Category,Coverage,Evaluation,Application,Further Tags
-"Jeho Oh, Paul Gazzillo, Don Batory, Marijn Heule, Maggie Myers",(techreport),2019,"Uniform Sampling from Kconfig Feature Models","","","","","","",""
-" Daniel-Jesus Munoz,  Jeho Oh,  Monica Pinto,  Lidia Fuentes,  Don Batory",SPLC,2019,"Uniform Random Sampling Product Configurations of Feature Models That Have Numerical Features","","","","","","",""
+"Juliana Alves Pereira, Mathieu Acher, Hugo Martib, Jean-Marc Jezequel",Proceedings of the ACM/Standard Performance Evaluation Corporation International Conference on Performance Engineering,2020,"Sampling Effect on Performance Prediction of Configurable Systems: A Case Study","","","","","effectiveness, available tool, open-source tool, evaluation","non-functional properties","compares Coverage-based/Solver-based/Randomized solver-based/Distance-based/Diversified distance-based/Random"
+" Sebastian Krieter,  Thomas Th&uuml;m,  Sandro Schulze,  Gunter Saake,  Thomas Leich",VaMoS,2020,"YASA: Yet Another Sampling Algorithm","YASA","feature model","greedy","t-wise coverage","sampling efficiency, testing efficiency, available tool, open-source tool, evaluation","testing","compared against Chvatal/ICPL/IncLing"
+" Quentin Plazar,  Mathieu Acher,  Gilles Perrouin,  Xavier Devroey,  Maxime Cordy",ICST,2019,"Uniform sampling of sat solutions for configurable systems: Are we there yet?","","","","","sampling efficiency, effectiveness, available tool, open-source tool, evaluation","testing, non-functional properties","compares QuickSampler/UniGen"
+" Alexandre Nuttinck Axel Halin, Mathieu Acher, Xavier Devroey, Gilles Perrouin, Benoit Baudry",Empirical Software Engineering,2019,"Test them all, is it worth it? Assessing configuration sampling on the JHipster Web development stack","","","","","testing efficiency, effectiveness, evaluation","testing, type checking","compares t-wise(SPLCAT)/one-disabled/one-enabled/most-enabled-disabled/dissimilarity(PLEDGE)/random, SPLC18"
```

Comment:
SPLC18 tags should not be added to new entries or removed from existing ones

Reply:
Please add a tag for all those that you have classified or modified: JS20 (short for thesis of Joshua Sprey 2020)

Reply:
Check

Reply:
Check